src/applications/personalization/profile-2/tests/components/Profile2Wrapper.unit.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { expect } from 'chai';
+
+import Profile2Wrapper from '../../components/Profile2Wrapper';
+import getRoutes from '../../routes';
+import { PROFILE_PATHS } from '../../constants';
+
+import { renderWithProfileReducers as render } from '../unit-test-helpers';
+import { Test } from 'mocha';
+
+describe('Profile2Wrapper', () => {
+  const config = {};
+  const ui = (
+    <MemoryRouter initialEntries={[PROFILE_PATHS.PERSONAL_INFORMATION]}>
+      <Profile2Wrapper routes={getRoutes(config)} />
+    </MemoryRouter>
+  );
+
+  it('should render the correct breadcrumb (Personal and contact Information)', () => {
+    const initialState = {
+      vaProfile: {
+        hero: {
+          userFullName: {
+            first: 'Test',
+            last: 'Test',
+          },
+        },
+      },
+    };
+    const { getByTestId } = render(ui, { initialState });
+    const breadcrumbs = getByTestId('breadcrumbs');
+    expect(breadcrumbs.textContent.match(/Personal and contact information/i))
+      .not.to.be.null;
+  });
+
+  it('should render ProfileHeader when the full name of the user was fetched)', () => {
+    const initialState = {
+      vaProfile: {
+        hero: {
+          userFullName: {
+            first: 'Test',
+            last: 'Test',
+          },
+        },
+      },
+    };
+    const { getByTestId } = render(ui, { initialState });
+    const profileHeader = getByTestId('profile-header');
+    expect(profileHeader.textContent.match(/Test Test/i)).not.to.be.null;
+  });
+
+  it('should not render ProfileHeader when the full name of the user could not be fetched)', () => {
+    const initialState = {
+      vaProfile: {
+        hero: {
+          errors: ['This is an error'],
+        },
+      },
+    };
+    const { queryByTestId } = render(ui, { initialState });
```

Comment:
I needed to use `queryByTestId` here because using `getByTestId` on a element that does not exists throws an error. https://dev.to/lek890/test-for-element-exists-react-testing-library-hg5

Reply:
yep, that's definitely the way to go! https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence